packages/apps/crm/src/utils/list.js

```diff
@@ -18,17 +18,6 @@
 
 import {isDate} from '@axelor/aos-mobile-core';
 
-export const getLastItem = (listItem, orderField) => {
-  return findItem(
-    listItem,
-    item => isDate(item[orderField]) && new Date(item[orderField]) < new Date(),
-    (prev, current) =>
-      new Date(current[orderField]) > new Date(prev[orderField])
-        ? current
-        : prev,
-  );
-};
-
 export const getNextItem = (listItem, orderField) => {
```

Comment:
If you are moving getLastItem, move getNextItem too as their logic is similar. Also you duplicated findItem